cibuildwheel/oci_container.py

```diff
@@ -108,7 +108,16 @@ def __enter__(self) -> OCIContainer:
         if detect_ci_provider() == CIProvider.travis_ci and platform.machine() == "ppc64le":
             network_args = ["--network=host"]
 
-        shell_args = ["linux32", "/bin/bash"] if self.simulate_32_bit else ["/bin/bash"]
+        simulate_32_bit = self.simulate_32_bit
```

Comment:
As written here, the parameter might be better named `ensure_32_bit`, as now it only does so when it's required.